frontend/src/layouts/header-layout.tsx

```diff
@@ -8,27 +8,30 @@ import { useEffect, useState } from 'react';
 import axios from 'axios';
 import { toast } from 'react-toastify';
 import userState from '@/utils/user-state';
+import { useCookies } from 'react-cookie';
 function header() {
   const navigate = useNavigate();
-
   const [accessToken, setAccessToken] = useState<string | null>(userState.getUser());
+  const [allCookies] = useCookies(['accessToken']);
 
-  useEffect(() => {
-    const storedAccessToken = userState.getUser();
+  setTimeout(() => {
+    setAccessToken(null);
+  }, 240 * 1000);
 
+  useEffect(() => {
+    const storedAccessToken = allCookies.accessToken;
     if (storedAccessToken) {
-      setAccessToken(storedAccessToken);
+      setAccessToken(allCookies.accessToken);
     }
```

Comment:
When we can directly get it from allCookies.accessToken, why are we storing it in acesstoken? And where is it being used